.travis.yml

```diff
@@ -13,6 +13,9 @@ matrix:
      os: linux
    # These don't have -dyn/-prof whitelisted yet, so we have to
    # do the old-style installation
+   - env: GHCVER=7.4.2 SCRIPT=script CABAL_LIB_ONLY=YES
+     os: linux
+     sudo: required
    - env: GHCVER=7.6.3 SCRIPT=script
```

Comment:
We discussed setting hard compatibility limits. Should cabal-install also stop supporting GHC 7.6?